01_todo_app/web/src/components/landing/index.jsx

```diff
@@ -1,68 +1,133 @@
-  }, []);
+    useEffect(() => {
+        dispatch(getMessages())
+    }, [])
 
-  return (
-    <div class="mui-container">
-      <div class="mui-panel mui--text-center">
-        <h1>My ToDo List</h1>
-        <h2>Application that allows you to store your tasks, based on a distributed architecture!</h2>
-      </div>
-      <div class="mui-panel mui--text-center">
-        <div class="form-wrapper">
-          <form class="mui-form--inline" onSubmit={addNewTodo}>
-            <div class="mui-textfield">
-              <input type="text" name="text" placeholder="Write something to do..."/>
-            </div>
-            <div class="mui-textfield">
-              <input type="hidden" value="NEW" name="status"/>
-            </div>
-            <button class="mui-btn mui-btn--primary mui-btn--raised">Add</button>
-          </form>
-        </div>
-        <div class="todo-list">
-          {messages.map(message => (
-            <div class="todo-item todo-item-{{ msg.status|lower }}">
-              <div class="mui-checkbox">
-                <label>
-                  <input type="checkbox" value="" name="checked" onChange={event => updateTodo(event, message)} checked={message.status === 'DONE'}/>
-                </label>
-              </div>
-              <span class="text">{ message.text }</span>
-              <form onSubmit={event => deleteTodo(event, message)}>
-                <button class="mui-btn mui-btn--danger mui-btn--raised">Remove</button>
-              </form>
+    return (
+        <>
+            <div class="mx-auto max-w-5xl px-4 sm:px-6 lg:px-8 mt-10">
+                <div className="text-center border-b border-gray-400 mb-10">
+                    <div className="inline-block text-white">
+                        <div className="flex items-center justify-center align-center my-10">
+                            <img className="h-14 rounded-full mr-4" src="todo-app.png" alt="Golem Logo" />
+                            <h1 className="text-3xl uppercase font-bold">
+                                ToDo <span className="font-light -ml-2 text-gray-300">List</span>
+                            </h1>
+                        </div>
+                        <div className="flex justify-center align-center my-10">
+                            <div className="md:w-1/2">
+                                <p className="block text-normal text-gray-300">
+                                    The ToDo list app is a demo showcasing the capability of running a simple app with a backend, frontend
+                                    and a database on the Golem Network. The purpose of this app is to demonstrate the possibility of
+                                    running these services on the network.
+                                </p>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+                <div>
+                    <div>
+                        <form
```

Comment:
Exercise to skill-up using components: Extract this to `TaskAddForm` component which should reside in it own file.